tests/Behat/Resources/services.xml

```diff
@@ -27,7 +27,7 @@
             <argument type="service" id="__symfony__.sylius.repository.order" />
             <argument type="service" id="__symfony__.sylius_refund.repository.refund" />
             <argument type="service" id="__symfony__.Sylius\RefundPlugin\Provider\RemainingTotalProvider" />
-            <argument type="service" id="__symfony__.prooph_service_bus.sylius_refund_command_bus" />
+            <argument type="service" id="__symfony__.message_bus" />
```

Comment:
Shouldn't it use the specific command bus / event bus?

Reply:
We can't, as only the default message bus (in fact it's alias `message_bus`) is public :|